config/examples/Robo3D/R1+/Configuration.h

```diff
@@ -1859,7 +1859,7 @@
  */
 
 // Min software endstops constrain movement within minimum coordinate bounds
-#define MIN_SOFTWARE_ENDSTOPS false // ncrmro
+#define MIN_SOFTWARE_ENDSTOPS
```

Comment:
This was set to `false`, but I didn't disable it here since the machine homes to MIN and does not have extra negative X/Y/Z travel.